pkg/loadbalancerservice/controller.go

```diff
@@ -209,3 +238,59 @@ func (c *LoadbalancerServiceController) patchStatus(svc *corev1.Service, newStat
 
 	return err
 }
+
+func (c *LoadbalancerServiceController) getRouterIPAddressList() ([]string, error) {
+	configuredAddresses, err := config.GetConfiguredAddresses()
+	if err != nil {
+		return nil, err
+	}
+
+	configuredNicNames, err := config.GetHostNICNames()
+	if err != nil {
+		return nil, err
+	}
+
+	ipList := make([]string, 0)
+
+	for _, ip := range c.IPAddresses {
+		if !slices.Contains(configuredAddresses, ip) {
+			klog.Infof("IP address %v not found in the host. Removing it", ip)
```

Comment:
The address is just being ignored, right, not removed? Will a reader of the log understand why we're reporting about that address at all?

Reply:
Depends on the context. These are the addresses that a user configured, and they were in the service status field for some time.
If somehow the IP disappears from the host then it is effectively removed from the service status. This may qualify better as a warning instead, as it points to a misconfiguration of MicroShift/the host.
It is true though that the removal only happens once. The next execution loop will not remove it because it was not there (provided that the situation persists). Will reword to skip and raise the level to warning.